awsenv/replacer_test.go

```diff
@@ -133,3 +184,87 @@ type mockParamsGetter func(context.Context, []string) (map[string]string, error)
 func (f mockParamsGetter) GetParams(ctx context.Context, paths []string) (map[string]string, error) {
 	return f(ctx, paths)
 }
+
+func TestFilterPaths(t *testing.T) {
+	tests := []struct {
+		prefix string
+		input  map[string]string
+		want   []string
+	}{
+		{
+			prefix: "",
+			input:  nil,
+			want:   []string{},
+		},
+		{
+			prefix: "awsenv:",
+			input:  map[string]string{"X": "1", "Y": "pre:/y", "Z": "awsenv:/z"},
+			want:   []string{"/z"},
+		},
+		{
+			prefix: "pre:",
+			input:  map[string]string{"X": "1", "Y": "pre:/y", "Z": "awsenv:/z"},
+			want:   []string{"/y"},
+		},
+	}
+
+	for _, test := range tests {
```

Comment:
Is there a reason we are not  using our standard subtest pattern (i.e `t.Run(...` in this PR?